playground/vue-3/index.html

```diff
@@ -16,7 +16,7 @@
       src="https://dhqbrvplips7x.cloudfront.net/common-ui-docs/genesys-webcomponents/3.59.0-715/genesys-webcomponents/genesys-webcomponents.esm.js"
     ></script>
 
-    <script src="https://unpkg.com/vue@next"></script>
+    <script src="https://unpkg.com/vue@^3.4.0"></script>
```

Comment:
vue@next has been removed in favor of vue@<version_number>. This is what was causing my issues because Vue could not be found to run createApp